plugins/telemetry-otel/src/main/java/org/opensearch/telemetry/tracing/sampler/package-info.java

```diff
@@ -0,0 +1,12 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+/**
+ * This package contains classes needed for HeadBased Sampler.
```

Comment:
Please remove mentions of HeadBased Sampler, all samplers  that we implement here are head based

Reply:
Remove it in newer comment.